google-cloud-bigquery/lib/google/cloud/bigquery/load_job.rb

```diff
@@ -1253,14 +1317,180 @@ def labels= val
           end
 
           ##
-          # Sets the partitioning for the destination table. See [Partitioned
+          # Sets the field on which to range partition the table. See [Creating and using integer range partitioned
+          # tables](https://cloud.google.com/bigquery/docs/creating-integer-range-partitions).
+          #
+          # See {#range_partitioning_start=}, {#range_partitioning_interval=} and {#range_partitioning_end=}.
+          #
+          # You can only set range partitioning when creating a table. BigQuery does not allow you to change
+          # partitioning on an existing table.
+          #
+          # @param [Integer] field The range partition field. the destination table is partitioned by this
```

Comment:
I think this type is supposed to be String, for the field name.

Reply:
Fixed, thanks!